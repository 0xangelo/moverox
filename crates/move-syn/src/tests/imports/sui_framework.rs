use super::*;

test_files!("sui-framework" {
    accumulator,
    address,
    authenticator_state,
    bag,
    balance,
    bcs,
    borrow,
    clock,
    coin,
    config,
    deny_list,
    display,
    dynamic_field,
    dynamic_object_field,
    event,
    hex,
    linked_table,
    math,
    object,
    object_bag,
    object_table,
    package,
    party,
    pay,
    priority_queue,
    prover,
    random,
    sui,
    table,
    table_vec,
    token,
    transfer,
    tx_context,
    types,
    url,
    vec_map,
    vec_set,
    versioned,
});
