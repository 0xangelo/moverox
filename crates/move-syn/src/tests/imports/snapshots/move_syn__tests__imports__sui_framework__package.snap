---
source: crates/move-syn/src/tests/imports/sui_framework.rs
expression: flat_imports(ast)
---
String -> std::ascii::String
type_name -> std::type_name
types -> sui::types
